src/Storage/StorageInterface.js

```diff
@@ -0,0 +1,37 @@
+goog.module('measurementlibrary.Storage.StorageInterface');
+
+/**
+ * A class implementing this interface allows for storage and retrieval of data.
+ *
+ * @interface
+ */
+class StorageInterface {
+  /**
+   * Saves a key value pair into the storage interface for up to a
+   * maximum time to live.
+   *
+   * @param {string} key The location at which to store data in the model.
+   *    Dot notation is used to access a nested value (i.e. 'employees.jim'
+   *    is the key 'jim' in the nested 'employees' object).
+   * @param {*} value The data to store.
+   * @param {number=} secondsToLive The maximum number of seconds that the key
+   *    should be saved for.
+   */
+  save(key, value, secondsToLive) {};
+
+  /**
+   * Loads the value associated with a given key in the storageInterface.
+   *
+   * @param {string} key The location at which to load data.
```

Comment:
Looks like key is listed twice in the JSdoc.

Reply:
Fixed